core/internal/clientstate/request-timeout.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2018 NEC Laboratories Europe GmbH.
```

Comment:
"2018-2019" or "2019"?

Reply:
This code was actually written in 2018. I'll update it to "2018-2019", then.